conf/application.conf

```diff
@@ -100,6 +100,23 @@ auditing {
   }
 }
 
+GovSpending {
+  GovSpendingWelfare = "22.1"
+  GovSpendingHealth = "20.5"
+  GovSpendingStatePensions = "12.4"
+  GovSpendingEducation = "11.6"
+  GovSpendingNationalDebtInterest = "6.9"
+  GovSpendingDefence = "5.3"
+  GovSpendingTransport = "4.3"
+  GovSpendingPublicOrderAndSafety = "4.3"
+  GovSpendingBusinessAndIndustry = "3.8"
+  GovSpendingGovernmentAdministration = "2.1"
+  GovSpendingHousingAndUtilities = "1.8"
+  GovSpendingCulture = "1.5"
+  GovSpendingEnvironment = "1.5"
+  GovSpendingOverseasAid = "1.1"
+  GovSpendingEuBudget = "0.8"
+}
```

Comment:
Shouldn't these 19/20 figures be visible from January. Till December should we show the 18/19 figures same way we are doing for treasury spending page?